### PR TITLE
Use rudder for deploy related functionality.

### PR DESCRIPTION
**Pull request by codeviking**
This ensures that the application is deployed to the latest n'
greatest skiff cluster.

### CODE REVIEW

File: .skiff/cloudbuild-deploy.yaml [hunk 1]
```diff
@@ -43,42 +43,14 @@ steps:
...
-  args: [
-    'get',
-    '-f', './webapp.json',
-    '-k', 'deployment',
-    '-o', './deployment.json'
-  ]
-  dir: '.skiff'
 # Deploy the image to Kubernetes
 - id: 'deploy'
-  name: 'gcr.io/cloud-builders/kubectl'
```
Comment (schmmd) [inline .skiff/cloudbuild-deploy.yaml]: Why `cloud-builders` vs. `ai2-reviz`?

Comment (codeviking) [reply .skiff/cloudbuild-deploy.yaml]: Fixed, it's `ai2-reviz` now. 

The `cloud-builders` are officially published by Google.  Rudder isn't there (yet!).

**Comment by codeviking** [comment]
@schmmd @joelgrus Of note.  The `rudder` command blocks until a deployment is complete.  Y'all have a lot of pods in your deployment.

Is this going to take a really long time, and/or cause problems? If so I may need to add a flag to rudder that lets apps opt out of blocking.

**Comment by schmmd** [comment]
Let's try and find out.

**Pull request merged**